src/test/functional/signin_user_test.ts

```diff
@@ -0,0 +1,29 @@
+const TestData = require('../test_data');
+
+Feature('User Signin');
+
+Scenario('I as an user can signin', ({ I }) => {
+  I.amOnPage('/login');
+  I.see('Sign in');
+  I.fillField('#username', 'idamOwner@hmcts.net');
+  I.fillField('#password', 'Pa55word11');
+  I.fillField('#username', TestData.SMOKE_TEST_USER_USERNAME);
+  I.fillField('#password', TestData.SMOKE_TEST_USER_PASSWORD);
```

Comment:
There are 2 sets of credentials here. Please remove one of them

Reply:
Fixed